module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -9,74 +9,97 @@
-| Base `R`          | <a href = "https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/Extract">`[[x]]`</a>   |  Double bracket       | Extracts `x` from a nested list                                |
-| Base `R`          | [`sessionInfo()`](https://www.rdocumentation.org/packages/utils/versions/3.5.1/topics/sessionInfo) | Session Information   | Returns the R version information, the OS, and the attacked packages in the current R session                                                                                                 |
-| Base `R`          | [`getwd()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/getwd)             | Get working directory | Finds the current working directory.                                                                                                                                                              |
-| Base `R`          | [`setwd()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/getwd)             | Set working directory | Changes the current working directory.                                                                                                                                                            |
-| Base `R`          | [`dir.exists()`](https://www.rdocumentation.org/packages/logcondens.mode/versions/1.0.1/topics/dir.exists)         | Directory exists      | Checks the file path to see if the directory exists there                                                                                                                                         |
+| Base `R`          | [`str(x)`](https://www.rdocumentation.org/packages/utils/versions/3.6.2/topics/str)               | Object Structure      | Gets a summary of the object `x` structure.                                                                                                                                                         |
+| Base `R`          | [`class(x)`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/class)                            | Object Class      | Returns the type of the values in object `x`.                                                                |
+| Base `R`          | [`nrow(x)`; `ncol(x)`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/nrow)                            | Number of Rows; Number of Columns    | Get the number of rows and the number of columns in an object `x`, respectively.                                                          |
+| Base `R`          | [`length(x)`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/length)                            | Number of Rows    | Returns how long the vector `x` is.                                                               |
```

Comment:
For more than just vectors. Could be matrix, list, etc. Just say `object` and leave at that.